code/AssetLib/Blender/BlenderIntermediate.h

```diff
@@ -116,7 +116,7 @@ namespace Blender {
         mywrap arr;
     };
 
-#ifdef _MSC_VER
+#if _MSC_VER < 1500
```

Comment:
Why 1500 here and not 1900? Possible typo?

Reply:
Correct, that is a typo. It should be 1900.